app/views/layouts/admin.html.erb

```diff
@@ -9,10 +9,20 @@
 <% end %>
 
 <% content_for :head do %>
+  <%= render "govuk_publishing_components/components/google_tag_manager_script", {
+    gtm_id: "GTM-P93SHJ4Z"
+  } %>
   <%= stylesheet_link_tag "admin_legacy.css", integrity: false %>
   <%= csrf_meta_tags %>
 <% end %>
 
+<% content_for :navbar do %>
+  <!-- Google Tag Manager (noscript) -->
+    <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-P93SHJ4Z"
+    height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
+  <!-- End Google Tag Manager (noscript) -->
+<% end %>
```

Comment:
If this is something we should track for Whitehall it's probably something we should do for other apps and have a component. I was going to suggest it should be part of the tag manager component, but I see about the difference in head or not. I do wonder though - we probably could just have the whole component at the start of the body element I guess and also include noscript.

Also as discussed face-to-face, I did wonder whether this noscript approach was something that was potentially an option as a way to help @patrickpatrickpatrick's [RFC](https://github.com/alphagov/govuk-rfcs/pull/160) where it suggests using ES5 for IE to still get analytics but it'd be so much better if they just got a lo-fi analytics approach like this. Though do wonder about cookie consent?

Reply:
Yes, have removed this for now as it's not needed for our initial testing but will consider the approach in future.